Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth
# Opening of Ulduar

## Changelog:
- Ulduar is closed again.
- You can send an expedition to open Ulduar.



# How to test:
Try to open Ulduar(you need to own land in Northrend and be Emperor) and make sure event work correctly.
